6. Developer Guides/Code Styleguide/README.md

```diff
@@ -4,4 +4,4 @@ order: 4
 
 # Rocket.Chat Code Styleguide
 
-- [Less Code Styleguide](less)
\ No newline at end of file
+- [Less Code Styleguide](https://github.com/Prakharsvnit/Rocket.Chat.Docs/blob/patch-1/6.%20Developer%20Guides/Code%20Styleguide/less.md)
```

Comment:
This is not the correct way, as it would just point to your repository and not the web page. Please correct it and make it like other links which are the correct way.